packages/plans-grid-next/src/components/storage/components/plan-storage-label.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+	PLAN_BUSINESS,
+	PLAN_ECOMMERCE,
+	PlanSlug,
+	StorageOption,
+} from '@automattic/calypso-products';
+import { AddOns } from '@automattic/data-stores';
+import { formatCurrency } from '@automattic/format-currency';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { usePlansGridContext } from '../../../grid-context';
+import useIsLargeCurrency from '../../../hooks/use-is-large-currency';
+import useStorageStringFromFeature from '../hooks/use-storage-string-from-feature';
+
+interface Props {
+	storageOption: StorageOption;
+	planSlug: PlanSlug;
+}
+
+const ELIGIBLE_PLANS_FOR_STORAGE_UPGRADE = [ PLAN_BUSINESS, PLAN_ECOMMERCE ];
```

Comment:
this needs to be revisited and centralized differently. it might actually just be redundant actually